CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [15.1.2] - 2020-02-24
```

Comment:
Leave the date as TBD, we'll set it when we do a release. We normally do minor version releases if we update libraries so go with 15.2.0 as the version number.